admin/app/controllers/workarea/admin/application_controller.rb

```diff
@@ -102,7 +102,7 @@ def setup_alerts
       end
 
       def track_index_filters
-        session[:last_index_path] = request.fullpath
+        session[:last_index_path] = request.fullpath unless request.format.json?
```

Comment:
i know this is technically more encompassing than `request.xhr?` but do we ever make json requests not from xhr? It would be more consistent with logic in other places.

Reply:
Agreed, when we talked about this at the sprint planning we said we should check both JSON and XHR and ignore for both.